app/controllers/admin/claims_controller.rb

```diff
@@ -29,4 +30,15 @@ def search
       flash.now[:notice] = "Cannot find a claim with reference \"#{params[:reference]}\""
     end
   end
+
+  private
+
+  def policy_from_param
+    @policy_from_param ||= begin
+      return nil if params[:policy].blank?
+
+      policy = Policies.all.map(&:to_s).find { |p| p == params[:policy].camelize }
```

Comment:
If we use `routing_name` we could potentially do:
```
policy = Policies[params[:policy]]
```

Reply:
Waaaaay ahead of ya - I just realised there was a simpler way of doing this 😄